src/app/shared/recommendation-card/recommendation-card.component.ts

```diff
@@ -8,10 +8,18 @@ import { Router } from '@angular/router';
 })
 export class RecommendationCardComponent  implements OnInit {
   @Input() cardData: any
+  imageFormats = ["jpg", "png", "jpeg", "gif", "tiff", "tif", "webp"]
```

Comment:
(optional) should we include allowed formats in a seperate config just in case if we have more formats or video as well if they're applicable